### ISSUE TITLE
Set `IS_TRAINING_APP` to true for unbranded 

### ISSUE THREAD

**Issue by mrjones-plip** [medic/cht-android#278]
**Describe the issue**
A number of deployments use the unbranded app for training. This app can be confused for the production app and there's no easy way to tell which is which. 

**Describe the improvement you'd like**
Now that 1.0.0 has the ability to [show a warning and red border](https://docs.communityhealthtoolkit.org/apps/guides/onboarding/#setting-up-a-training-app), we should add this to the unbranded app.

**Comment by mrjones-plip** [comment]
@latin-panda - would this count as a 1.0.1 release? Maybe not because it's just a gradle file change?

**Comment by latin-panda** [comment]
If you want to publish in PlayStore then _I think_ we need a new version, otherwise it won't let you upload APKs with same version.

**Comment by mrjones-plip** [comment]
1.0.1 it is then!  We for sure want it in the play store ASAP

thanks!

**Comment by mrjones-plip** [comment]
> Do you get a yellow and a red border?

I agree we need to check for what happens, but unless the app is totally unusable, I don't think it should block this feature from being released ASAP.  Based on [my research](https://forum.communityhealthtoolkit.org/t/best-practices-for-data-integrity-during-on-boarding/1779/4), I think the common use case is to use the unbranded app for training (`*.dev.medicmobile.org`) or developing (`*.my.local-ip.co`).

We have a partner that wants to use the app in training on a in a week or two on a `*.dev.medicmobile.org` domain and would greatly benefit from this feature being live!

**Comment by abbyad** [comment]
Adapting some of my comments from a Slack conversation about this issue... 

Although the unbranded app may be used for training, it is not exclusively for training. The `IS_TRAINING_APP` flag will put the [training toast](https://github.com/medic/cht-android/blob/d8051e7ee6d7c1222b5f29aec7cc2f81858744af/src/main/res/values/strings.xml#L56) for all users of the unbranded app -- including those using it for partner demos and meetings. It's also possible that an unknown (non-Medic) deployment of CHT is using the unbranded app, although I am not aware of that being the case.

Having the border red for production use of [app.medicmobile.org](http://app.medicmobile.org/) makes sense to me, but I don't understand why the unbranded app should have the IS_TRAINING_APP flag. Can someone provide more clarity on that? 

Another option is to change the toast for this flavor to be more generic, like `This app is not production use`. Either way, we should focus on border color and message for the unbranded app, and the implementation details can be sorted out later.
